<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-06-15T19:51:30 IST" name="Default suite" finished-at="2024-06-15T19:51:49 IST" duration-ms="18781">
    <groups>
    </groups>
    <test started-at="2024-06-15T19:51:30 IST" name="Default test" finished-at="2024-06-15T19:51:49 IST" duration-ms="18781">
      <class name="com.opencart.testLayer.LoginTest">
        <test-method is-config="true" signature="start()[pri:0, instance:com.opencart.testLayer.LoginTest@4567f35d]" started-at="2024-06-15T19:51:30 IST" name="start" finished-at="2024-06-15T19:51:31 IST" duration-ms="362" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- start -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.opencart.testLayer.LoginTest@4567f35d]" started-at="2024-06-15T19:51:31 IST" name="setUp" finished-at="2024-06-15T19:51:40 IST" duration-ms="9039" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="verifyLoginTestWithInvalidDetails()[pri:0, instance:com.opencart.testLayer.LoginTest@4567f35d]" started-at="2024-06-15T19:51:40 IST" name="verifyLoginTestWithInvalidDetails" finished-at="2024-06-15T19:51:41 IST" duration-ms="1683" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLoginTestWithInvalidDetails -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.opencart.testLayer.LoginTest@4567f35d]" started-at="2024-06-15T19:51:41 IST" name="tearDown" finished-at="2024-06-15T19:51:42 IST" duration-ms="567" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.opencart.testLayer.LoginTest@4567f35d]" started-at="2024-06-15T19:51:42 IST" name="setUp" finished-at="2024-06-15T19:51:47 IST" duration-ms="4829" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="verifyLoginTestWithValidDetails()[pri:0, instance:com.opencart.testLayer.LoginTest@4567f35d]" started-at="2024-06-15T19:51:47 IST" name="verifyLoginTestWithValidDetails" finished-at="2024-06-15T19:51:48 IST" duration-ms="1442" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [My Account] but found [Account Login]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [My Account] but found [Account Login]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.opencart.testLayer.LoginTest.verifyLoginTestWithValidDetails(LoginTest.java:33)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLoginTestWithValidDetails -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.opencart.testLayer.LoginTest@4567f35d]" started-at="2024-06-15T19:51:48 IST" name="tearDown" finished-at="2024-06-15T19:51:49 IST" duration-ms="738" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="end()[pri:0, instance:com.opencart.testLayer.LoginTest@4567f35d]" started-at="2024-06-15T19:51:49 IST" name="end" finished-at="2024-06-15T19:51:49 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- end -->
      </class> <!-- com.opencart.testLayer.LoginTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
